src/components/CommentsCards.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
+import { getCommentsByReviewId } from '../utils';
+
+const CommentsCards = ({ review_id}) => {
+  const [commentsList, setCommentsList] = useState([]);
+  const [commentsLoading, setCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setCommentsLoading(true)
+    getCommentsByReviewId(review_id).then((commentsFromApi) => {
+      setCommentsList(commentsFromApi);
+      setCommentsLoading(false);
+    });
+  }, []);
+
+  console.log(commentsList);
+
+  if (commentsLoading) {
```

Comment:
Love the seperate is loading for the comments so the user isnt waiting for both the article and comments to come back from the api before they can see anything